repos/versions/v1.0.0.repos

```diff
@@ -0,0 +1,45 @@
+repositories:
+  dave/dave:
+    type: git
+    url: https://github.com/Field-Robotics-Lab/dave.git
+    version: v1.0.0
+
+  dave/gtri_based_sonar:
+    type: git
+    url: https://github.com/Field-Robotics-Lab/gtri_based_sonar.git
+    version: 5778a8008c52da4abf672046e3c38d15649c5eb3
```

Comment:
It might also be worthwhile to create tags for these commits where possible, such as the dave-specific repositories.